cloud_level_testing/features/steps/definitions.py

```diff
@@ -601,3 +605,29 @@ def attach_floating_ip_to_server(context, server_name):
         server = context.client.compute.find_server(name_or_id=server_name)
         assert server, f"Server with name {server_name} not found"
         ip = context.client.add_auto_ip(server=server, wait=True)
+        context.vm_ip_address = ip
+        context.logger.log_info(f"Attached floating ip: {ip}")
+    
+    @then('I start calculating 4000 digits of pi on VM and check the ping response')
+    def calculate_pi_on_vm(context):
+        """
+        """
+        # config
+        ping_retries = 60
```

Comment:
can we move that to some class constant?

Reply:
Do you wanna have it defined in the features or just a constant?

Reply:
just a constant